athletics.lic

```diff
@@ -290,6 +302,8 @@ class Athletics
         start_script('performance', ['noclean']) unless Script.running?('performance')
         pause @performance_pause # make sure you are playing before starting climb practice
       end
+      move("climb #{target}") if target2
```

Comment:
Possibly make this an array and just `move climb` for each if this is the best solution.